.github/workflows/continuous-integration.yaml

```
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        node-version: [10, 12, 14, 15]
+    services:
+      mysql:
+        image: mysql:5.7.22
+        env:
+          MYSQL_USER: user
+          MYSQL_PASSWORD: pass
```

Maybe we can try to use the same values we have in place now?

```suggestion
          MYSQL_USER: test
          MYSQL_PASSWORD: test
```